examples/simple-color/agent.py

```diff
@@ -25,13 +28,22 @@ async def _draw_color():
         argb_frame = bytearray(WIDTH * HEIGHT * 4)
         while True:
             await asyncio.sleep(0.1)  # 100ms
-
-            argb_frame[:] = COLOR * WIDTH * HEIGHT
-            frame = rtc.VideoFrame(WIDTH, HEIGHT, rtc.VideoBufferType.RGBA, argb_frame)
+            # Generate random color values for R, G, B
+            r = random.randint(0, 255)
+            g = random.randint(0, 255)
+            b = random.randint(0, 255)
+
+            # Create a new random color
+            color = bytes([r, g, b, 255])
+
+            # Fill the frame with the new random color
+            argb_frame[:] = color * WIDTH * HEIGHT
+            frame = rtc.VideoFrame(
+                WIDTH, HEIGHT, rtc.VideoBufferType.RGBA, argb_frame)
             source.capture_frame(frame)
 
     await _draw_color()
 
 
 if __name__ == "__main__":
-    cli.run_app(WorkerOptions(entrypoint_fnc=entrypoint))
+    cli.run_app(WorkerOptions(entrypoint_fnc=entrypoint, agent_name="simple-color"))
```

Comment:
This is awesome! definitely less confusing!
Can you remove the agent_name inside the WorkerOptions, adding agent_name currently requires manual dispatch of agents and this part isn't well documented yet

Reply:
sure, just removed at [43dcc62](https://github.com/livekit/agents/pull/714/commits/43dcc6285b52fc2bee61f837d4b0fca5b59f70e4)